modules/openapi-generator/src/main/resources/JavaSpring/beanValidation.mustache

```diff
@@ -1,5 +1,7 @@
 {{#required}}
-  @NotNull
+    {{^vendorExtensions.x-snt-validation-message-not-blank}} {{^vendorExtensions.x-snt-validation-message-not-empty}} @NotNull {{#vendorExtensions.x-snt-validation-message-not-null}}(message = "{{{vendorExtensions.x-snt-validation-message-not-null}}}"){{/vendorExtensions.x-snt-validation-message-not-null}}       {{/vendorExtensions.x-snt-validation-message-not-empty}} {{/vendorExtensions.x-snt-validation-message-not-blank}}
```

Comment:
@arturobernalg thanks for the PR. What does "snt" stand for?

Reply:
HI @wing328 ...it's a mistake.i will change it

Reply:
change it. thanks